canonical-web-and-design/maas-ui
# feat(ui): single-col tag form

## Done

- Update the tag form to appear in a single column. See: https://www.figma.com/file/AFw4WN623gie2Fi34btFlZ/%5B22.04%5D-Tag-management?node-id=833%3A19723

## QA

### MAAS deployment

To run this branch you will need access to one of the following MAAS deployments:

- [Bolla](/HACKING.md#bolla)
- [A development MAAS](/HACKING.md#development-deployment)
- [A local snap MAAS](/HACKING.md#snap-deployment) (this will not usually have machines)

### Running the branch

You can run this branch by:

- Serving with [dotrun](/HACKING.md#maas-ui-development-setup)
- [Building in a development MAAS](/HACKING.md#running-maas-ui-from-a-development-maas)

### QA steps

- Go to the machine list, select some machines that have tags and open the tag form.
- The form should appear with the new design.

## Fixes

Fixes: canonical-web-and-design/app-tribe#776.

## Screenshot

<img width="1466" alt="Screen Shot 2022-03-29 at 2 02 05 pm" src="https://user-images.githubusercontent.com/361637/160528884-6976ec91-457d-4927-8732-cd8004140ff3.png">
